tests/test_accessibility.py

```diff
@@ -14,71 +14,32 @@
 from .pages import AccessibilityPage
 
 
-class NoAccessibilityTest(WebAppTest):
+class AxsAccessibilityTestMixin(object):
```

Comment:
nit: It would be easier to understand the test code and coverage if the mixin names were normalized. 
E.g. GoogleAxsTestMixin and AxeCoreTestMixin for BOKCHOY_A11Y_RULESET values google_axs and axe_core.